informer/src/informer_service/informer_service_handler.rs

```diff
@@ -98,3 +121,68 @@ impl InformerService {
         }
     }
 }
+
+pub struct BlockCreatedEventGenerator {
+    client: Arc<Client>,
+    rate_limiter: Limiter,
+    send_status: ColdEventSendStatus,
+}
+
+impl BlockCreatedEventGenerator {
+    pub fn new(client: Arc<Client>) -> Self {
+        Self {
+            client,
+            rate_limiter: ratelimit::Builder::new().quantum(1).interval(Duration::new(0, 5000)).build(),
+
+            send_status: ColdEventSendStatus::InProgress,
+        }
+    }
+
+    pub fn run(mut self, connection: Connection, mut favorite_block_number: u64) {
+        println!("I am here 3");
+        let rt = Runtime::new().unwrap();
```

Comment:
You should not use multiple runtimes if you don't have any special purpose on it.
Please use only one runtime for the Informer service.

Reply:
I do not use multiple run time. it is the only place that I used `runtime`.

Reply:
The position is problem.
If we want to add `TransactionMinedEventGenerator`, where will be the runtime?
If we follow the current structure, there will be multiple runtimes; one runtime for one generator.